docker-registry-show.py

```diff
@@ -39,8 +39,14 @@ def __init__(self):
         self.parser.add_argument('--username', metavar='USERNAME')
         self.parser.add_argument('--password', metavar='PASSWORD')
 
-        self.parser.add_argument('--authorization-service', metavar='AUTH_SERVICE', type=str,
```

Comment:
let's keep backwards compatibility by adding a deprecated `--authorization-service` that automatically fills in the authorization_service_url and authorization_service_name

Reply:
sure, that's a good idea. will do